9-ObjectCapabilities/README.md

```diff
@@ -62,7 +62,7 @@ So far all of our factory methods have required us to pass in names on which to
 Another technique is to allow the factory to create the necessary unforgeable names and send them back to the caller. I call this the a "full service" factory. If you don't require the flexibility of passing in arbitrary names, a full service factory is often less hassle.
 
 ### Exercise
-Convert one idiom to the other
+Convert the savings account from byoc factory to to full service factory.
```

Comment:
This is a good change, but it doesn't belong in a PR that is about translation. I've cherry-picked this commit, so it doesn't need to be in this PR.